.github/workflows/configurable-benchmark-test.yaml

```diff
@@ -36,12 +36,12 @@ jobs:
     if: ${{ github.event_name == 'workflow_dispatch' }}
     steps:
       - name: Setup Kubernetes
-        uses: manusa/actions-setup-minikube@v2.7.1
+        uses: medyagh/setup-minikube@v0.0.8
         with:
           minikube version: v1.23.2 
           kubernetes version: v1.23.2
           driver: docker
-          start args: --disk-size 10000mb
+          memory: 5120m

```

Comment:
Why the change to a different workflow?

Reply:
I'll revert it, no issues spotted here actually